supabase/config.toml

```diff
@@ -0,0 +1,178 @@
+# A string used to distinguish different Supabase projects on the same host. Defaults to the
```

Comment:
Can we add instructions in the README for how to start / stop the DB for local development?

Reply:
Added API.md here: https://github.com/polywrap/AutoTx/pull/254/files#diff-b161966f7254af9c9c5382e859495c69622843e4602a6bb4208e08134e660121
And liked to it from the main README.md